arcgis-android-toolkit/build.gradle

```diff
@@ -55,6 +56,9 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    dataBinding {
```

Comment:
This actually works without databinding enabled at the moment.

Reply:
No. We need dataBinding enabled to be able to use DatabindingUtil and ViewDataBinding classes in BookmarkAdapter.

https://github.com/Esri/arcgis-runtime-toolkit-android/pull/146/files#diff-6972c309ea40b7e84309ccee57d3091cR106

https://github.com/Esri/arcgis-runtime-toolkit-android/pull/146/files#diff-6972c309ea40b7e84309ccee57d3091cR135